bundles/com.espressif.idf.sdk.config.core/src/com/espressif/idf/sdk/config/core/KConfigMenuProcessor.java

```diff
@@ -84,7 +84,10 @@ protected void read(Object obj, KConfigMenuItem menuItem)
 			childMenu.setId((String) jsonObject.get("id")); //$NON-NLS-1$
 			childMenu.setType((String) jsonObject.get("type")); //$NON-NLS-1$
 			childMenu.setHelp((String) jsonObject.get("help")); //$NON-NLS-1$
-
+			if (jsonObject.get("is_menuconfig") != null) {
+				childMenu.setIsMenuConfig((boolean) jsonObject.get("is_menuconfig")); //$NON-NLS-1$
+			}
```

Comment:
We can create private static constant variable for these json keys. Or you can use the //$NON-NLS-1$ and //$NON-NLS-2$ if there are more than two hardcoded strings in one statement

Reply:
done